File: PharmaDicBackEnd/PharmaDicBackEnd.ApiService/DTOs/SymptomInputDto.cs
using System.ComponentModel.DataAnnotations;

namespace PharmaDicBackEnd.ApiService.DTOs
{
    public class SymptomInputDto
    {
        [Required(ErrorMessage = "Tên triệu chứng không được để trống.")]
        [StringLength(100, ErrorMessage = "Tên triệu chứng không được vượt quá 100 ký tự.")]
        public string SymptomName { get; set; } = null!;

        public string? Description { get; set; }
    }
}

File: PharmaDicMobile/app/src/main/java/com/example/pharmadicmobile/dtos/SymptomInputDto.java
package com.example.pharmadicmobile.dtos;

public class SymptomInputDto {
    private String symptomName;
    private String description;

    public SymptomInputDto() {}

    public String getSymptomName() { return symptomName; }
    public void setSymptomName(String symptomName) { this.symptomName = symptomName; }

    public String getDescription() { return description; }
    public void setDescription(String description) { this.description = description; }
}
